packages/office-ui-fabric-react/src/components/Calendar/CalendarPage.tsx

```diff
@@ -24,6 +24,10 @@ export class CalendarPage extends React.Component<IComponentDemoPageProps, any>
               <CalendarInlineExample isMonthPickerVisible={ false } dateRangeType={ DateRangeType.Day } autoNavigateOnSelection={ false }
                 showGoToToday={ false } />
             </ExampleCard>
+            <ExampleCard title='Inline Calendar with overlayed month picker when header is clicked' code={ CalendarInlineExampleCode }>
+              <CalendarInlineExample isMonthPickerVisible={ true } showMonthPickerAsOverlay={ true } highlightCurrentMonth={ true } dateRangeType={ DateRangeType.Day } autoNavigateOnSelection={ false }
```

Comment:
Yes. I think it was just left over from a copy and paste of another example.  I will remove.